torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -101,7 +100,58 @@ def ones(
     Returns:
         A :class:`ShardedTensor` object on each rank
     """
-    tensor_init_params = TensorInitParams(create_op=CreateOp.ONES, dtype=dtype, layout=layout,
+    tensor_init_params = TensorInitParams(create_op=CreateOp.ONES, fill_value=0,
+                                          dtype=dtype, layout=layout,
+                                          requires_grad=requires_grad,
+                                          pin_memory=pin_memory, memory_format=memory_format)
+    return ShardedTensor(
+        sharding_spec,
+        *size,
+        tensor_init_params=tensor_init_params,
+        process_group=process_group,
+        init_rrefs=init_rrefs,
+    )
+
+
+def rand(sharding_spec: ShardingSpec,
+         *size,
+         dtype=None,
+         layout=torch.strided,
+         requires_grad=False,
+         pin_memory=False,
+         memory_format=torch.contiguous_format,
+         process_group=None,
+         init_rrefs=False):
+    """
+    Creates a rand :class:`ShardedTensor`. Needs to be called on all ranks in an SPMD fashion.
```

Comment:
I think we can copy some of the docs from `torch.rand()` here (see https://pytorch.org/docs/stable/generated/torch.rand.html):

```
Returns a tensor filled with random numbers from a uniform distribution on the interval [0, 1)[0,1)
```

Reply:
Good point, done.